### PR TITLE
Increase max value in SearchLabel widget

### PR DESCRIPTION
**Pull request by clemenshug**
Hi everyone,

Thank you for creating this amazing piece of software!

I noticed that the maximum value for the search label widget is 999. Given that cell labels tend to be much larger than that would it make sense to increase the maximum? I couldn't find a way to make it unbounded but I figured a sensible max would be the largest value uint64 can encode.

What do you think?

### CODE REVIEW

File: package/PartSeg/plugins/napari_widgets/search_label_widget.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def __init__(self, napari_viewer: Viewer):
         self.napari_viewer = napari_viewer
         self.search_type = create_widget(annotation=SearchType, label="Search type")
         self.search_type.changed.connect(self._component_num_changed)
-        self.component_selector = SpinBox(name="Label number", min=0)
+        self.component_selector = SpinBox(name="Label number", min=0, max=2**64 - 1)
```
Comment (Czaki) [inline package/PartSeg/plugins/napari_widgets/search_label_widget.py:23]:
```suggestion
        self.component_selector = SpinBox(name="Label number", min=0, max=2**31 - 1)
```

Qt int has 4 bits https://bugreports.qt.io/browse/QTBUG-63765 and is unsigned int, so There is a need to lower the max value.

**Review by Czaki — COMMENTED** [review]
Hi @clemenshug, 
Thanks for this suggestion. I agree that limit is too low, but currently, Qt does not support soo big ints. Is the limit from my suggestion enough for you?

**Comment by Czaki** [comment]
@clemenshug ?

**Comment by clemenshug** [comment]
Hi @Czaki and thank you for checking!
I originally put in 1e8 for testing but then increased it further for the PR without testing again. Sorry about that.
2**31 - 1 it is!

**Comment by Czaki** [comment]
Thanks. I assume that next week I will release a new version.

**Pull request merged**